mava/utils/jax_training_utils.py

```diff
@@ -44,34 +44,53 @@ def construct_norm_axes_list(
     start_axes: int, axes_list: List[Any], obs_shape: Tuple
```

Comment:
Can you rename `axes_list` to `elements_to_norm` and make the type `List[Union[Int],Tuple[Int,Int]]` (depending on if you decide to keep or remove the extra case for tuples inside the list)

Also can you remove `start_axis` and make it the first element of `elements_to_norm`

Reply:
we can't remove start_axis. start_axis is an information that comes from the environment. This is not provide by the user. Also I tried modify the type to be more explicit but this just leads to so many cryptic error messages from mypy.